components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -361,11 +361,11 @@ - (void)removeExistingViewControllers {
 }
 
 /**
- Adjusts all relevant insets in subviews and the selected child view controller. This include @c
- safeAreaInsets and scroll view insets.  This will ensure that although the child view controller's
- view is positioned behind the bar, it can still lay out its content above the Bottom Navigation
- bar.  For a UIScrollView, this means manipulating both @c contentInset and
- @c scrollIndicatorInsets.
+ * Adjusts all relevant insets in subviews and the selected child view controller. This include @c
```

Comment:
Tip: I'd honestly prefer for this file to move toward the project's convention. Leading asterisks on lines are at odds with most of the library. That being said, this file has them for most things.

Reply:
Ah ok.  I changed the comments in the controller and related files to match this style.